contrib/generic_proxy/filters/network/source/proxy.cc

```diff
@@ -50,7 +50,8 @@ ActiveStream::ActiveStream(Filter& parent, StreamRequestPtr request)
     : parent_(parent), request_stream_(std::move(request)),
       request_stream_end_(request_stream_->frameFlags().endStream()),
       stream_info_(parent_.time_source_,
-                   parent_.callbacks_->connection().connectionInfoProviderSharedPtr()) {
+                   parent_.callbacks_->connection().connectionInfoProviderSharedPtr(),
+                   StreamInfo::FilterState::LifeSpan::FilterChain) {
```

Comment:
All changes that to add a `LifeSpan::FilterChain` argument is safe because the original default value is `LifeSpan::FilterChain`